my_list = [2, 4, 6, 7]
my_list[3] = 8

print(my_list)

#using get method for unfacing error

print("using get method")
my_nums = {1 : 1, 2 : 4, 3 : 9, 4 : 16}
print(my_nums.get(5, "5 not in my numbers!"))
